### PR TITLE
Arm updates

### PR DESCRIPTION
**Pull request by AlisonDuan**
Updates for running our object detection code in conjunction with a precision arm.

### CODE REVIEW

File: inf_od_nwk.py [hunk 1]
```diff
@@ -23,28 +23,35 @@
 
 import jetson.inference
 import jetson.utils
+
+#  this is a test comment
```
Comment (BrandonMan123) [inline inf_od_nwk.py:27]:
```suggestion
```

File: inf_od_nwk.py [hunk 2]
```diff
@@ -133,24 +140,45 @@
             cv2.circle(depth_img, (int(clamp_x2), int(clamp_y2)), 5, (0, 0, 255), -1)
             cv2.imshow("clamp points", depth_img)
             key = cv2.waitKey(0)
-
+            if key & 0xFF == ord('q') or key == 27:
```
Comment (BrandonMan123) [inline inf_od_nwk.py:143]: Should probably add print statement to tell the user what to do 
```suggestion
            print ("Press q to quit or r to continue")
            if key & 0xFF == ord('q') or key == 27:
```

File: src/arm/publish_arm_updates.py [hunk 3]
```diff
@@ -90,18 +89,41 @@ def writeToSerial(writeArray):
 		second_half = max(angle-255, 0)
 		split_write_array.append(first_half)
 		split_write_array.append(second_half)
+<<<<<<< HEAD
```
Comment (BrandonMan123) [inline src/arm/publish_arm_updates.py:92]: Is this supposed to be here?

File: src/arm/publish_arm_updates.py [hunk 4]
```diff
@@ -90,18 +89,41 @@ def writeToSerial(writeArray):
...
-    ''' This function publishes updates to the arm configuration by writing 
-        to the serial buffer. We use R2Protocol to write to the arduino, which 
-        will connect/write to the motor controllers directly. 
+def publish_updates(update_array, timeout):
+	''' 
+	This function publishes updates to the arm configuration by writing 
+	to the serial buffer. We use R2Protocol to write to the arduino, which 
+	will connect/write to the motor controllers directly. 
 
+<<<<<<< HEAD
```
Comment (BrandonMan123) [inline src/arm/publish_arm_updates.py:109]: Same as comment above

**Review by BrandonMan123 — COMMENTED** [review]
Everything seems good to me except for the files I talked about

File: src/arm/publish_arm_updates.py [hunk 5]
```diff
@@ -1,29 +1,34 @@
...
 import numpy as np
 
 import serial
 import sys
 import codecs
+import time 
+import math
 
 # Dedicate a /dev port for specifically the precise arm
+<<<<<<< HEAD
```
Comment (BrandonMan123) [inline src/arm/publish_arm_updates.py:12]: Merge conflict here

File: src/arm/publish_arm_updates.py [hunk 6]
```diff
@@ -119,8 +141,9 @@ def publish_updates(updates, timeout):
         writeToSerial(update_array)
         print("array {} sent: {}".format(index, update_array))
         time.sleep(timeout)
+>>>>>>> master
```
Comment (BrandonMan123) [inline src/arm/publish_arm_updates.py:144]: Conflict here

File: src/kinematics/linear_rrt.py [hunk 7]
```diff
@@ -229,8 +230,13 @@ def radians_to_degrees(rrtnode):
     return degrees
 
 
+<<<<<<< HEAD:src/kinematics/assuming_linearity_rrt.py
+def path_radians_to_degrees(path: List[RRTNode]):
+    """Converts a path of RRTNode instances into a path of degree arrays to be used in arm encoder movements."""
+=======
```
Comment (BrandonMan123) [inline src/kinematics/linear_rrt.py:236]: Merge conflict here

**Review by SimonK5 — APPROVED** [review]

**Review by DaiYijia02 — COMMENTED** [review]
The edits looks reasonable!

**Pull request merged**